packages/openapi/src/index.ts

```diff
@@ -182,9 +182,9 @@ const customFormats = {
   }
 }
 
-interface ValidationError {
+export interface ValidationError {
   status?: number
-  errors: string[]
+  errors: ReturnType<typeof errorTransformer>[]
```

Comment:
The return type of `errorTransformer` is 

```ts
{
  message: string
}
```

Might it make sense to define that and use it here and the `errorTransformer` signature?

Reply:
Yup, I think I went with this since I didn't have a good name for the type 😆

Reply:
Notice that I didn't offer a name myself lol